prometheus_client/registry.py

```diff
@@ -106,6 +115,25 @@ def collect(self):
 
         return RestrictedRegistry()
 
+    def set_target_info(self, labels):
+        with self._lock:
+            if labels:
+                if not self._target_info and 'target_info' in self._names_to_collectors:
+                    raise ValueError('CollectorRegistry already contains a target_info metric')
+                self._names_to_collectors['target_info'] = None
+            elif self._target_info:
+                self._names_to_collectors.pop('target_info', None)
```

Comment:
Since the result of `.pop(...)` is not used, does it make sense to not pass an argument at all for that param rather than `None`?